Doc/library/calendar.rst

```diff
@@ -138,23 +138,54 @@ interpreted as prescribed by the ISO 8601 standard.  Year 0 is 1 BC, year -1 is
 
    :class:`TextCalendar` instances have the following methods:
 
+   .. method:: formatday(theday, weekday, width=2)
```

Comment:
The `width=2` suggested to me that one can pass this a keyword argument, but that is not the case. Also there does not seem to be a default value for the `TextCalendar` and the `weekday` argument is not used at all. 

https://github.com/python/cpython/blob/b9a492b809d8765ee365a5dd3c6ba4e5130a80af/Lib/calendar.py#L342-L350

I tried looking at the unit tests to see what the method actually does, but I cannot find any.